src/components/Loader/Loader.tsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
```

Comment:
Consider adding a short description comment explaining the purpose of the Loader component 📝